infra/modules/database/dms-task-cdc.tf

```diff
@@ -0,0 +1,148 @@
+# docs: https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/dms_replication_task
+resource "aws_dms_replication_task" "replication_task_cdc" {
+  replication_instance_arn = aws_dms_replication_instance.instance.replication_instance_arn
+  source_endpoint_arn      = aws_dms_endpoint.source_endpoint.endpoint_arn
+  target_endpoint_arn      = aws_dms_endpoint.target_endpoint.endpoint_arn
+  migration_type           = "cdc"
+  replication_task_id      = "${var.environment_name}-cdc"
+  # The following settings are our modifications from the default settings.
+  # Terraform will merge these settings with the default settings.
+  #
+  # Because we don't explicitly specify all ~200 of the default settings,
+  # terraform will show a diff that includes all of the default settings.
+  # You can safely ignore that diff.
+  #
+  # Example diff here: https://betagrantsgov.slack.com/archives/C05TSL64VUH/p1707427008889219
+  replication_task_settings = jsonencode(
+    {
+      "Logging" : {
+        "EnableLogging" : true,
+        "EnableLogContext" : true,
+      },
+      "ValidationSettings" : {
+        "EnableValidation" : true,
+      },
+    }
+  )
+  table_mappings = jsonencode(
```

Comment:
The CDC task and the full load task are using different `table_mappings` because at some point I was having them write to different tables. Exclusively for debugging purposes. Since the CDC task isn't working yet, I'm probably going to bring that back in the future.

At any rate, that's why `table_mappings` is copy pasted here, rather than being assigned to a variable.